working-with-your-data/working-with-your-own-data.md

```diff
@@ -114,6 +118,70 @@ wget --user=<USERNAME> --ask-password <URL>
 ```
 Using the `--ask-password` will prompt you to enter your password.
 
+### Upload large files (> 1Gb) from your own computer
+
+#### Install FileZilla
+
+Go to [FileZilla's website](https://filezilla-project.org/download.php?type=client) to download the FileZilla Client for your respective operating system.
+
+Click the big green `Download` button.
+
+Click `Download` on this next page for `FileZilla` this is the only free option but will have the functionality you need.
+
+<img src="screenshots/filezilla-download.png">
+
+After download is complete, you'll find the `FileZilla_` file in your download files or you can click on it in the corner of your web browser's screen.
+On a Mac, this will be a a `.app.tar.bz2`, on a Windows, it will be a `.exe` file. 
```

Comment:
```suggestion
On a Mac, this will be a `.app.tar.bz2` file. On Windows, it will be a `.exe` file. 
```

Reply:
I think keeping Mac and Windows instructions in brief separate sections (under a bold line **Mac** if you don't want it to be included in the TOC) could be useful here, rather than having them kinda blended together.

Reply:
Yup. I was kinda thinking about that, but now that you've said, it I will do that.